src/commands/check.js

```diff
@@ -54,10 +55,19 @@ export const handler = catchAsyncError(async opts => {
   // Checking all the deps if all of them are omitted
   if (_.every(DEPS_GROUPS, ({name}) => !opts[name])) {
     _.each(DEPS_GROUPS, ({name}) => (opts[name] = true));
+    opts.global = false;
+  }
+
+  // Make global flag mutually exclusive with other flags
+  else if (opts.global) {
+    for (let i = 0; i < DEPS_GROUPS.length; i++)
+      opts[DEPS_GROUPS[i].name] = false;
```

Comment:
Let's better use the same code as above:
```suggestion
    _.each(DEPS_GROUPS, ({name}) => { opts[name] = false });
```

Reply:
I tried copying this line but it simply wouldn't do what I asked of it. I don't know what lodash does with it in the background but the options would never be set to `false` for some reason.

Reply:
That's because when you write it as `_.each(DEPS_GROUPS, ({name}) => opts[name] = false)` result of `opts[name] = false` expression is `false` and lodash cancels iteration after the first call. That's why you need to write it like that: `_.each(DEPS_GROUPS, ({name}) => { opts[name] = false })` as result of `{ opts[name] = false }` is `undefined` and lodash will iterate through all items.